gui/build.py

```diff
@@ -118,19 +113,48 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
+    if IS_WIN:
+        default_python_path = 'C:\\Python27\\Lib'
+        bundle.add_argument('--nsis-path', dest='nsis_path',
+                            default='C:\\NSIS',
+                            help='your NSIS path on the'
+                            ' system(default: %(default)r)')
+    else:
+        default_python_path = '/usr/lib/python2.7'
+
     bundle.add_argument('--python-path', dest='python_path',
```

Comment:
please use '--python-lib-path' here. also the dest option is not needed (same thing for nsis_path option): https://docs.python.org/3/library/argparse.html#dest